Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1634,4 +1632,20 @@ namespace AtomSampleViewer
             });
     }
 
+    AZ::RHI::MultisampleState SampleComponentManager::CreateDefaultMsaaState() const
+    {
+        AZ_Assert(IsValidNumMSAASamples(m_numMSAASamples), "Invalid MSAA sample setting");
+        const AZ::RHI::SamplePosition Center = { 8, 8 };
+        const AZ::RHI::SamplePosition TopLeft = { 0, 0 };
+
+        AZ::RHI::MultisampleState state;
+
+        state.m_samples = static_cast<uint16_t>(m_numMSAASamples);
+        state.m_customPositionsCount = 2;
+        state.m_customPositions[0] = Center; 
+        state.m_customPositions[1] = TopLeft;
```

Comment:
Can you fill in all 16 positions with the list I gave you? That way someone can easily switch MSAA back to a different count and all will still work.

Reply:
Done